libraries/3rdparty/mbedtls_config/aws_mbedtls_config.h

```diff
@@ -9,7 +9,13 @@
  */
 /*
  *  Copyright (C) 2006-2018, ARM Limited, All Rights Reserved
- *  SPDX-License-Identifier: Apache-2.0
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
```

Comment:
Why is this config file not part of every vendor project instead of a single file under 3rdparty just like other config files for customer visibility of control of the mbedtls library?